src/util-server.ts

```diff
@@ -3,6 +3,7 @@
 	Do not import this file from a hydrated client-side component.
 */
 
+// @ts-expect-error Package without types we can’t do anything about.
```

Comment:
It's usually seen as better to declare an empty module for the package than to expect error it, but both are okay

`declare module "@11ty/eleventy-fetch";`

Reply:
My thinking was to limit this to the one file where we’re using eleventy-fetch. Declaring a module has to be done in a separate `.d.ts` otherwise, right?

Declaring a module in this file throws:

> Invalid module name in augmentation. Module '@11ty/eleventy-fetch' resolves to an untyped module, which cannot be augmented.

Reply:
Yeah, it would have to be in a separate file. `@ts-expect-error` is totally fine too